src/api/include/migraphx/migraphx.hpp

```diff
@@ -979,6 +979,23 @@ struct program : MIGRAPHX_HANDLE_BASE(program)
         return arguments(pout, own{});
     }
 
+// Needed to enable stream sync functionality at compile time with onnxruntime
+// Helps deal with compatability so we dont break execution provider.
+#define MIGRAPHX_STREAM_SYNC
```

Comment:
We should remove this, onnxrt should just check for the function or migraphx version.

Reply:
It allows for onnxruntime to compile based on the migraphx. Do we want to gate this by version then on the ORT side then and it select which lamda to use at runtime?

Reply:
I guess I can use MIGRAPHX_VERSION_MAJOR/MINOR? Is that something we want to expose in our API so we can do version checks for features at runtime then?

Reply:
I just removed this and added a hook in cmake for the API to pass on instead.